tv_apps/browser/index.html

```diff
@@ -1,364 +1,742 @@
+    <script defer="" src="js/search_util.js"></script>
+    <script defer="" src="js/authentication_dialog.js"></script>
+    <script defer="" src="js/browser_dialog.js"></script>
+    <script defer="" src="js/toolbar.js"></script>
+    <script defer="" src="js/tooltip.js"></script>
+    <script defer="" src="js/search_result.js"></script>
+    <script defer="" src="js/settings.js"></script>
   </head>
 
-  <body role="application">
-
-    <div id="main-screen" class="awesomescreen">
-
-      <div id="toolbar">
-        <button id="back-button" class="icon" disabled="true" alt="Back"></button>
-        <button id="forward-button" class="icon" disabled="true" alt="Forward"></button>
-        <button id="bookmark-button" class="icon" alt="Bookmark" role="button"></button>
-        <button id="zoom-button" class="icon" alt="Zoom" role="button"></button>
-        <form id="url-bar" novalidate>
-          <input type="image" id="ssl-indicator" value="" />
-          <input type="text" id="url-input" placeholder="Search or enter address"
-                 data-l10n-id="enter-search-or-address" x-inputmode="verbatim"
-                 dir="auto" />
-          <input type="image" id="url-button" value="" />
-        </form>
-        <button id="home-button" class="icon" alt="Home" role="button"></button>
-        <button id="tv-button" class="icon" alt="TV" role="button"></button>
-        <button id="pointer-button" class="icon" alt="Pointer" role="button"></button>
-        <button id="tabs-button" class="icon" alt="Tabs" role="button">6</button>
-        <button id="settings-button" class="icon" alt="Settings" role="button"></button>
-        <button id="awesomescreen-cancel-button"></button>
-        <div id="throbber"></div>
-      </div>
+  <body class="page_screen start_page" role="application">
+    <!-- mask for menu -->
+    <div id="fade-base"></div>
 
-      <div id="awesomescreen">
+    <!-- side -->
+    <div id="side-block" class="side-window">
+      <!-- put widget here -->
+    </div>
 
-        <div id="search-results-panel" class="is-hidden">
-        </div>
-        <nav data-role="tabs" id="tab-headers">
-          <ul role="tablist" class="bb-tablist">
-            <li role="presentation">
-              <a id="top-sites-tab" href="#" class="selected"
-                 data-l10n-id="top-sites-tab" role="tab"
-                 aria-selected="true" aria-controls="top-sites-panel">Top Sites</a>
+    <!-- main -->
+    <div id="main-block" data-mode="cursor" class="main-window">
+      <!-- toolbar -->
+      <ul id="toolbar-panel" data-pvt="false">
+        <div id="private-bar"></div>
+        <li id="sidebar-button-block" data-tips="WB_LT_TIPS_CHANGE_SCREEN" data-fade="false">
+          <div id="sidebar-button" data-sidebar="true">
+            <svg id="sidebar-button-svg" xmlns="http://www.w3.org/2000/svg" version="1.1" viewBox="3 3 47 47" x="0" y="0" preserveAspectRatio="xMidyMid slice">
```

Comment:
Let's add `XXX: Replace the SVG to icon font` for each SVGs embedded inside index.html